client/src/components/Sidebar/ChatContent.js

```diff
@@ -18,14 +19,20 @@ const useStyles = makeStyles((theme) => ({
     color: "#9CADC8",
     letterSpacing: -0.17,
   },
+  unreadCounter: {
+    alignItems: "flex-end",
+    padding: theme.spacing(10),
+  },
 }));
 
 const ChatContent = (props) => {
   const classes = useStyles();
 
   const { conversation } = props;
   const { latestMessageText, otherUser } = conversation;
-
+  // finds the length of unread messages sent by the other user
+  const unreadMessagesCount = conversation.messages.filter(message => message.read === false 
+    && message.senderId === otherUser.id).length;
```

Comment:
Since you're now counting messages inside a reducer function you don't need to have this logic here, feel free to remove it :)

Reply:
done